src/status_im2/contexts/wallet/common/temp.cljs

```diff
@@ -22,7 +22,9 @@
    [quo/button {:on-press #(rf/dispatch [:navigate-to :wallet-create-account])}
     "Create Account"]
    [quo/button {:on-press #(rf/dispatch [:navigate-to :wallet-saved-addresses])}
-    "Saved Addresses"]])
+    "Saved Addresses"]
+   [quo/button {:on-press #(rf/dispatch [:navigate-to :wallet-send-input-amount])}
```

Comment:
@briansztamfater - using temporary navigation for the moment - that's correct right or there's a possible navigation path?

Reply:
If we get to merge #17579 first, a possible (but still temporary until #17780 is also merged) navigation path would be navigating when clickin continue on the address selection page by using `:navigate-to-within-stack` instead of `:navigate-to`

Reply:
@briansztamfater 
For `:navigate-to-within-stack` what `component` should I use?
Or may be I can use `:open-modal`?
![Simulator Screen Recording - iPhone 11 Pro - 2023-11-07 at 19 58 40](https://github.com/status-im/status-mobile/assets/10757633/73174b29-2d1d-4224-a07b-79931bd60a43)

Reply:
Closed